### DISCUSSION TITLE
Adoption

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MoneroFlags#2] [Announcements]
Greetings @QuickBASIC (Mike Justman)!

I am asking for your permission to update / continue / adopt / release the below listed and named wonderful mod(s); unless somebody did already.

I absolutely love these mods and really want others to enjoy them as well.

Hopefully I hear back from you with an answer, but in case I don't I intend to respectfully proceed as soon as the middle of September 2021, as permitted by the licenses of these mods. I will provide recognition of original authors/maintainers; and I will gladly step aside should you ever want to continue this mod yourself.

Couple of requests:

I am asking for a Github post giving me permission along with tagging [ @DasSkelett ](https://forum.kerbalspaceprogram.com/index.php?/profile/188118-dasskelett/) and [ @HebaruSan ](https://forum.kerbalspaceprogram.com/index.php?/profile/156657-hebarusan/) so a smooth transition on CKAN which will allow update / release without confusing end users by keeping the CKAN listing intact. Links are included.

Any unreleased updates would be very helpful as well.

Thank you for your time and consideration!

-----
### MoneroFlags
- CC BY-SA 4.0 ![Creative Commons Attribution Share-Alike 4.0 Licence](https://camo.githubusercontent.com/0df8fd3e955d97ae69dedfa2568fb2dd4186cd60917ca40aefabb7466d5b46ce/68747470733a2f2f692e6372656174697665636f6d6d6f6e732e6f72672f6c2f62792d73612f342e302f38387833312e706e67)
- https://github.com/QuickBASIC/MoneroFlags
- CKAN: MoneroFlags
-----

**Comment by QuickBASIC** [comment]
Yes. Thank you for offering to continue to maintain these mods.  I currently have very limited time to play KSP and therefore don't have time to continue to maintain these.
